components/shared/isSSRRender.js

```diff
@@ -0,0 +1,3 @@
+const isSSRRender = () => typeof window === 'undefined';
```

Comment:
just suggestion: 

```suggestion
const isSSR = () => typeof window === 'undefined';
```

:crab: